protos/compile.sh

```diff
@@ -0,0 +1,9 @@
+#!/usr/bin/env bash
```

Comment:
I would like to keep script files in a `scripts` folder.

Reply:
Thanks for the suggestion! I prefer keeping build scripts colocated with their inputs for easier maintenance and to ensure they're always in sync. Do you have any thoughts?

Reply:
I don't think there is anything right or wrong about this. I prefer to put scripts in their folders because I can open the folder and immediately know what to do in the command line with the associated files. This works better if the codebase is large, and I'm more used to this way. I don't have a strong preference for this project, we could just align the existing practice.

Reply:
Yeah, I'll do that. I don't have a strong preference on this either